pkgs/by-name/do/docfd/package.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, ocamlPackages, fetchFromGitHub, python3, dune_3, makeWrapper, poppler_utils }:
+
+ocamlPackages.buildDunePackage rec {
+  pname = "docfd";
+  version = "2.1.0";
+
+  minimalOcamlVersion = "5.1";
```

Comment:
```suggestion
  minimalOCamlVersion = "5.1";
```

Typo?

Reply:
> Typo?

Many thanks for that! That was a tricky one to spot.